app/imports/components/Projects/CreateNewProject/CreateNewProject.jsx

```diff
@@ -1,26 +1,67 @@
 import React, { Component, PropTypes } from 'react';
 import { TextField,
     RaisedButton,
+    AutoComplete,
+    List,
+    ListItem,
 } from 'material-ui';
 
 class CreateNewProject extends Component {
 
     constructor(props) {
         super(props);
         this.onSubmit = this.onSubmit.bind(this);
+        this.addMods = this.addMods.bind(this);
         this.state = {
             isError: false,
             isResult: false,
+            usersNames: [],
+            usersIds: [],
```

Comment:
Instead having two array with separated names and ids, please make one array of objects:
```js
{
 name: ...,
 id: ...,
}
```
When you have object you always know user and his id. Array items can be shuffled.